docs/astro.config.mjs

```diff
@@ -199,6 +200,10 @@ export default defineConfig({
 					autogenerate: { directory: 'reference' },
 				},
 			],
+			plugins: [starlightLinksValidator({
+			  errorOnFallbackPages: false,
+			  errorOnInconsistentLocale: true,
+			})],
```

Comment:
Can we move this behind an environment variable flag and add a dedicated CI job that runs it? Right now a broken link causes many tests and the preview deployment to fail. It would be nice to separate it out so we have a clear indication that a broken link is the cause (can can still see the preview while waiting for the fix).

Something like:

```suggestion
			plugins: process.env.CHECK_LINKS ? [starlightLinksValidator({
			  errorOnFallbackPages: false,
			  errorOnInconsistentLocale: true,
			})] : [],
```

And then a new job in `.github/workflows/ci.yml` like:

```yml
  links:
    name: Check for broken links
    runs-on: ubuntu-20.04
    steps:
      - name: Checkout
        uses: actions/checkout@v3

      - name: Setup PNPM
        uses: pnpm/action-setup@v2

      - name: Setup Node
        uses: actions/setup-node@v3
        with:
          node-version: ${{ env.NODE_VERSION }}
          cache: 'pnpm'

      - name: Install Dependencies
        run: pnpm i

      - name: Build docs site and check for links
        working-directory: ./docs
        run: pnpm build
        env:
          CHECK_LINKS: true
```

Reply:
(Heh, I’m too slow for @HiDeoo as usual, but always on the same page 😁)